packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -24,13 +25,45 @@ interface GoogleCalError extends Error {
   code?: number;
 }
 
+const ONE_MINUTE_MS = 60 * 1000;
+const CACHING_TIME = ONE_MINUTE_MS;
+
+/** Expand the start date to the start of the month */
+function getTimeMin(timeMin: string) {
+  const dateMin = new Date(timeMin);
+  return new Date(dateMin.getFullYear(), dateMin.getMonth(), 1, 0, 0, 0, 0).toISOString();
+}
```

Comment:
```suggestion
function getStartDateOfTheMonth(dateStart: string) {
  const dateMin = new Date(dateStart);
  return new Date(dateMin.getFullYear(), dateMin.getMonth(), 1, 0, 0, 0, 0).toISOString();
}
```

Reply:
Will keep timeMin/Max to match what gCal is expecting.